vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -72,9 +72,14 @@ switch -Regex ($windowsSku) {
         )
     }
     "23H2*" {
-        # Don't need to update patchUrls and patchIDs for 23H2 since it's by design.
-        $global:patchUrls = @()
-        $global:patchIDs = @()
+        if ($windowsSku -eq "23H2-gen2") {
+            # Don't need to update patchUrls and patchIDs for 23H2 since it's by design.
```

Comment:
this comment should be for lines 80-81 ?

Reply:
The command is for windows security patch feature. I think that we can remove it now.